packages/yew-macro/src/stringify.rs

```diff
@@ -75,26 +78,31 @@ impl Stringify for LitStr {
         }
     }
 }
+
 impl Stringify for Lit {
     fn try_into_lit(&self) -> Option<LitStr> {
-        let s = match self {
+        let mut buf = [0; size_of::<char>()];
+        let s: Cow<'_, str> = match self {
             Lit::Str(v) => return v.try_into_lit(),
-            Lit::Char(v) => v.value().to_string(),
-            Lit::Int(v) => v.base10_digits().to_string(),
-            Lit::Float(v) => v.base10_digits().to_string(),
-            Lit::Bool(_) | Lit::ByteStr(_) | Lit::Byte(_) | Lit::Verbatim(_) => return None,
-            _ => unreachable!("unknown Lit"),
+            Lit::Char(v) => (&*v.value().encode_utf8(&mut buf)).into(),
+            Lit::Int(v) => v.base10_digits().into(),
+            Lit::Float(v) => v.base10_digits().into(),
+            Lit::Bool(v) => if v.value() { "true" } else { "false" }.into(),
+            Lit::ByteStr(v) => String::from_utf8(v.value()).ok()?.into(),
+            Lit::Byte(v) => v.value().to_string().into(),
+            Lit::Verbatim(v) => v.to_string().into(),
```

Comment:
Verbatim usually means that (this version of) syn does not understand what literal this is.
In this case, it is better to reject the literal.